test/In.ProjectEKA.HipServiceTest/DataFlow/DataFlowMessageHandlerTest.cs

```diff
@@ -13,6 +13,8 @@ namespace In.ProjectEKA.HipServiceTest.DataFlow
 
     public class DataFlowMessageHandlerTest
     {
+        private readonly string curve = "curve25519";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'curve'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5073772)